src/08-card/index.js

```diff
@@ -185,7 +185,7 @@ registerBlockType("nhsblocks/card1", {
 				<span role="text">
 				<span className="nhsuk-u-visually-hidden">Non-urgent advice: </span>
```

Comment:
Question className? Also, is it semantic to have a role="heading" inside a heading?